tools/release/release.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  releaseChangelog,
+  releasePublish,
+  releaseVersion,
+} from 'nx/src/command-line/release';
```

Comment:
[Product Feedback] Importing from a `src/` sub-path feels off to me. It'd be nice to have a more official/not-feeling-like-I'm-importing-from-internals way.

Reply:
I don’t disagree! Hopefully it’s something we can tweak later in the year

Reply:
For this and other product feedback points, is there a good link we can put in a todo comment?

Reply:
I've tracked it on our internal Linear, we don't tend to duplicate things like this into the OSS issue tracker. Feel free to open issues there any time or just ping me, either is fine

Reply:
👍 https://github.com/nrwl/nx/issues/21029